release/ray_release/test.py

```diff
@@ -501,11 +505,27 @@ def get_test_results(
             key=lambda file: int(file["LastModified"].timestamp()),
             reverse=True,
         )[:limit]
-        self.test_results = _asyncio_thread_pool.submit(
-            lambda: asyncio.run(
-                self._gen_test_results(bucket, [file["Key"] for file in files])
-            )
-        ).result()
+        if use_async:
+            self.test_results = _asyncio_thread_pool.submit(
```

Comment:
why `asyncio` does not work on windows?

Reply:
i'm not sure whether it's asyncio or because of the way we are using asyncio; i plan to migrate the entire code base to async gradually